README.md

```diff
@@ -1 +1,60 @@
-# Apache Spark Connect Client for Swift language
+# Apache Spark Connect Client for Swift
+
+[![GitHub Actions Build](https://github.com/apache/spark-connect-swift/actions/workflows/build_and_test.yml/badge.svg)](https://github.com/apache/spark-connect-swift/blob/main/.github/workflows/build_and_test.yml)
+
+This is an experimental Swift library to show how to connect to a remote Apache Spark Connect Server and run SQL statements to manipulate remote data.
+
+So far, this library project is tracking the upstream changes like the [Apache Spark](https://spark.apache.org) 4.0.0 RC2 release and [Apache Arrow](https://arrow.apache.org) project's Swift-support.
+
+## Requirement
+- [Apache Spark 4.0.0 RC2 (March 2025)](https://dist.apache.org/repos/dist/dev/spark/v4.0.0-rc2-bin/)
```

Comment:
It's better to be https://spark.apache.org/downloads.html or https://spark.apache.org/news/spark-4.0.0-preview2.html

Reply:
In that case, let me remove this section.